### DISCUSSION TITLE
Collectibles or *something* unique to incentivize returning to earlier moons

### DISCUSSION THREAD

**Discussion by Schurches** [Malcolm-Q/LC-LateGameUpgrades#549] [Ideas]
First of, thanks for creating this mod! It's amazing and it has added TONS of hours of gameplay for my friends and me.

**Idea**: Add some sort of collectibles, objectives, events, or something that may feel sort of unique for every moon (at the very least to the vanilla moons). This may give some incentive to go back to earlier moons even if the quota is not too high, or at least delay the desire to head over to late-game moons as soon as you get the $ to do so. A few ideas I've thought about are:
- Collectibles: It could be either a unique scrap or something that we can pick up (such as the logs) and that we could find on the moon. _In my head_ this could shake up a few runs as it would now encourage us to explore the moon looking for these collectibles instead of entering the facility right away. You could have multiple of them, like 3 for smaller maps such as Assurance and 5 for bigger maps like March.
- Objectives: Have a set of objectives (like a checklist) to do in a moon, such as finding a specific scrap, kill a specific monster, collect X amount of scrap, or carry out a contract mission. 
- Events: Allow the moons to have certain boosts from time to time (at random), such as X2 in scrap value, X2 or X3 the amount of scrap as it normally would (depending on the moon), or have the time pass slower (or faster) in a specific day due to some anomaly. I know that the last one is essentially what the quantum disruptor does, but it would now also be introduced as an "event" and not only locked behind an upgrade :)

The way to implement them could be to unlock them through the LGU store or have as a stand-alone section similar to the contracts. It could even require a set of criteria for us to unlock it just so we don't start using them right away, such as:
- Landing on the moon X amounts of times
- Collected a total of $X in scrap value
- Collected a total of X scraps
Again, all in order to incentivize landing on a moon more than once!

Finally, collecting all of them could either unlock a discount to any of the LGU upgrades, or reward the team with an item (Suit, normal item, decoration for the ship), or either allow the players to "do something" to boost a moon in some way (either permanently or temporarily) or just have the moon permanently boosted from then on. And of course, giving the hosts the option to customize some variables of the mod would be amazing, as well as have it compatible with other mods such as the facility meltdown, but all of that can wait!

**Reason**: So in all of our playthroughs, once we get to the very late game moons such as Titan, Artifice, or any cool custom moon, there's pretty much no need to go back to earlier moons as the high quota makes it pretty much mandatory for us to stay in those moons. It would be nice to have an incentive to return to earlier moons, especially if there's a reward for it.
I've been sitting on this idea for weeks. I hope I'm not asking for too much or isn't over complicated :D

**Comment by Schurches** [comment]
Too complicated I guess?

**Reply by WhiteSpike** [reply]
You essentialy asked for a mod which provides boosts based on your progress on a given moon which must check what items it can be spawned, what enemies it can spawn and if they are killable, check if the amount of scrap value does not exceed the total spawned for that level, check how many times you have landed in a given level.
Then after all the required checks for correct behaviour, you want that that it may give a suit which we would have to have to check if you didn't already purchased the suit, a normal item we would have to check if it's not an unintended item (vanilla has binoculars) and deco check if you didn't already purchase it. The boost part on moons through scrap amount/value would have to be another manager which manages the boosts on each moon (both perm and temp) and necessary steps for a boost.

So it's not a matter of complicated, it's a matter of complexity that you're asking to somebody who's not going to have a lot of free time due to having important things to do such as finishing a computer science masters degree. That's why I kept this here incase other people wanted to give it a try of doing this til I had enough free time to do anything productive.
